pkg/asset/agent/installconfig.go

```diff
@@ -214,9 +214,9 @@ func (a *OptionalInstallConfig) validateControlPlaneConfiguration(installConfig
 	var fieldPath *field.Path
 
 	if installConfig.ControlPlane != nil {
-		if *installConfig.ControlPlane.Replicas != 1 && *installConfig.ControlPlane.Replicas != 3 {
+		if *installConfig.ControlPlane.Replicas != 1 && *installConfig.ControlPlane.Replicas != 3 && *installConfig.ControlPlane.Replicas != 4 && *installConfig.ControlPlane.Replicas != 5 {
```

Comment:
```suggestion
		if *installConfig.ControlPlane.Replicas < 1 || *installConfig.ControlPlane.Replicas > 5 ||*installConfig.ControlPlane.Replicas == 2 {
```
Slightly shorter comparison